apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js

```diff
@@ -26,10 +27,6 @@ function LaunchWpcomWelcomeTour() {
 		isManuallyOpened: select( 'automattic/wpcom-welcome-guide' ).isWelcomeGuideManuallyOpened(),
 	} ) );
 
-	const localeSlug = useLocale();
-	// Preload first card image (others preloaded after open state confirmed)
-	new window.Image().src = getTourSteps( localeSlug )[ 0 ].meta.imgSrc;
```

Comment:
> Preload first card image (others preloaded after open state confirmed)

Do we no longer want this optimization?

Reply:
Hmm, not sure what happened here (probably some rebasing stuff) but I reverted the changes.